rest-openehr/src/main/java/org/ehrbase/rest/openehr/OpenehrCompositionController.java

```diff
@@ -331,20 +340,18 @@ public ResponseEntity getComposition(
 
         URI uri = createLocationUri(EHR, ehrId.toString(), COMPOSITION, versionedObjectUid);
 
-        List<String> headerList = Arrays.asList(
-                LOCATION,
-                ETAG,
-                LAST_MODIFIED); // whatever is required by REST spec - CONTENT_TYPE only needed for 200, so handled
-        // separately
+        // whatever is required by REST spec - CONTENT_TYPE only needed for 200, so handled separately
+        var headerList = List.of(LOCATION, ETAG, LAST_MODIFIED);
```

Comment:
We usually only use var if generics make it extremely unreadable, as it is hard to review